products/ceqr_survey/models/staging/stg__pluto.sql

```diff
@@ -6,11 +6,11 @@ WITH mappluto_wi AS (
 
 final AS (
     SELECT
-        CAST(bbl AS text) AS bbl,
         zonedist1,
         zonedist2,
         zonedist3,
         zonedist4,
+        CAST(bbl AS text) AS bbl,
```

Comment:
Can we move this back? I assume this was from `sqlfluff fix` being run not from the home directory?